src/assets/styles/_mixin.scss

```diff
@@ -33,6 +33,8 @@
     'light-color': $light-background,
     'primary-color': $primary,
     'l-blue': $light-blue,
+    'l-grey': $light-grey,
+    'smoke-bg': $white-smoke,
   );
 
   @each $color in $colors {
```

Comment:
sorry forgot to refactor this my-self haha. since they are modifier change .text-#{$color} to `.color--#{$color}` and  .bg-#{$color} to `.bg--#{$color}` nam. then refactor all the classes. just use ctrl+shift+f and replace it